rest/rest-api/src/main/java/org/ow2/proactive_grid_cloud_portal/common/SchedulerRestInterface.java

```diff
@@ -2413,4 +2413,41 @@ JobValidationData validateJobSignal(@HeaderParam("sessionid") String sessionId,
     @Produces(MediaType.APPLICATION_JSON)
     Map<String, Map<String, Boolean>> checkJobsPermissionMethods(@HeaderParam("sessionid") String sessionId,
             PermissionForm multipart) throws RestException;
+
+    /**
+     *
+     * Add an external endpoint url to the endpoint list of the considered job
+     *
+     * @param sessionId
+     *            current session
+     * @param jobId
+     *            id of the job
+     * @param externalEndpointUrl
+     *          the external endpoint url to add to the job endpoint list
+     */
+    @POST
+    @Path("job/{jobid}/externalendpointurls")
+    @Consumes(MediaType.APPLICATION_JSON)
+    @Produces(MediaType.APPLICATION_JSON)
+    void addExternalEndpointUrl(@HeaderParam("sessionid") String sessionId, @PathParam("jobid")
+    final String jobId, String externalEndpointUrl) throws RestException;
+
+    /**
+     *
+     * Remove an external endpoint url from the endpoint list of the considered job
```

Comment:
Remove an external endpoint url from a job. The job must be alive and not terminated, otherwise an UnknownJobException will be thrown.

Reply:
Ok